lib/fb/page.rb

```diff
@@ -60,10 +72,15 @@ def to_s
 
   private
 
-    def insights(metric, options = {})
-      params = options.merge metric: metric, access_token: @access_token
+    def get_daily_metrics(metric, options = {})
```

Comment:
@S1v4 This method is not _really_ return daily metrics. In fact, you add `period: :day` to the options when you invoke this method inside `view_count`, but you don't inside `like_count`.

Instead, this method gives you a way to access an _individual_ metric. All the other methods allow for multiple metrics to be provided; this method only accepts one metric. I would rename to `individual_metric` or something on that line.